.github/SUPPORT.md

```diff
@@ -0,0 +1,38 @@
+# Support
+
+Thank you for using Erlang LS! :tada:
+
+This document offers some guidelines when asking for support related to an issue
+you might be having with the Erlang LS tool.
+
+We will always try to help as much as we can. This is an open-source project and
```

Comment:
Not sure about this sentence, since it almost suggests to expect limited support and that open-source means based-on-free-time. I would just drop it.

Reply:
I can drop it, but the point here is to point out that people shouldn't expect free cutomer service.